### PR TITLE
Hero implementation

### LINKED ISSUE #25: Hero/Banner Component
Please refer to the design: https://xd.adobe.com/spec/967e129d-f3bb-4a08-6f39-553da7ddef6d-e04f/

### PR DESCRIPTION
**Pull request by franreysaycon**
**Description**
Implemented the hero component

**Summary of Changes**
- Made images modular, so I don't have to create a longer relative path.
- Created hero component
- Added hero fontWeight in `themes.js` ( we only have 400 and 900, but in the design "we are" is 600 )
- Added thin fontWeight in `themes.js` (for the button)

**Screenshots**
image

**Notes**
This is a work in progress as I am waiting for the new hero image as we can't handle clip paths using the given brandmark.

Closes #25  - issue number

### CODE REVIEW

**Comment by ericsoncluciano** [comment]
lgtm 👌🏼

File: src/components/hero.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { Box, Text, Button } from "rebass"
+import { css } from "@emotion/core"
+import { heroImg, heroBrandmark } from "../images"
+import Container from "./container"
+
+const heroStyle = css`
+    position: relative;
+    left: 50%;
```
Comment (psongpin) [inline src/components/hero.js]:
```suggestion
```

Comment (psongpin) [inline src/components/hero.js]:
```suggestion
  
```

File: src/components/hero.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    right: 50%;
+    background: url("${heroImg}") no-repeat center;
+    width: 100vw;
+    height: 80rem;
+    margin-left: -50vw;
+    margin-right: -50vw;
```
Comment (psongpin) [inline src/components/hero.js]: I would suggest removing this wrapper box `<Box m="0 auto" maxWidth={960} px={1}>` in layout.js then remove `left`, `right`, `margin-left` and `margin-right` properties. My main concern for that wrapping jsx is it limits the flexibility of the layout and having us resort to negative margins to achieve full width components. We could just stack containers if needed.

**Comment by psongpin** [comment]
@franreysaycon @ericsoncluciano @jromest I adjusted some styles to fix the z-index of the layers in the hero. here is how it looks right now.

Screen Shot 2019-10-11 at 9 09 46 PM

**Pull request merged**